tools/flaky-test-reporter/slack_notification.go

```diff
@@ -84,7 +84,8 @@ func sendSlackNotifications(repoDataAll []RepoData, c slackutil.WriteOperations,
 		for i := range channels {
 			wg.Add(1)
 			channel := channels[i]
-			go func(wg *sync.WaitGroup) {
+			go func() {
```

Comment:
are you sure this works without passing wg?

Reply:
Yes, one example here - https://golang.org/src/sync/example_test.go